tidy.cfg

```diff
@@ -0,0 +1,23 @@
+indent: auto
+indent-spaces: 2
+tab-size: 2
+wrap: 132
+markup: yes
+output-xml: no
+input-xml: no
+numeric-entities: yes
```

Comment:
I tend to prefer raw Unicode for printable characters, but okay

Reply:
I agree. Removing this line.